### DISCUSSION TITLE
非常感谢作者！

### DISCUSSION THREAD

**Discussion by HCY-ASLEEP** [1943time/inkdown#58] [General]
天哪，我找了一堆开源md编辑器，就这个好用啊，我只是想打开指定目录编辑md, 然后上传 github page, 结果试了一堆，要不是要 npm 安装，就是要创建什么笔记，崩溃啊！比如说 Zettlr 并没有给黑暗模式做适配，Joplin 预览的代码块好像改不了字体

**Comment by 1943time** [comment]
我的初衷就是想做个好看易书写的markdown编辑器，同时可以自由的分享，谢谢你的认可😄。

**Comment by Tribbtsz** [comment]
简洁易用，功能强大，收下我的膝盖
